JS/app.js

```diff
@@ -1,7 +1,18 @@
-
+ let counter = 0;
```

Comment:
the use of counter variable seems like adding more to lines of code, better juts checking if element display is block or none